server/libs/domain/src/media/media.service.ts

```diff
@@ -93,6 +95,40 @@ export class MediaService {
     return true;
   }
 
+  async handleGenerateThumbhashThumbnail({ id }: IEntityJob) {
+    const [asset] = await this.assetRepository.getByIds([id]);
+    if (!asset || !asset.resizePath) {
+      return false;
+    }
+    try {
+      const image = sharp(asset.resizePath);
+      const resizedImage = image.resize(100,100);
```

Comment:
If an image is less than 100x100. What would happen here?

Reply:
It would upscale. 

I really should change that to respect aspect ratio.